splice_machine/src/main/java/com/splicemachine/derby/utils/Vacuum.java

```diff
@@ -84,27 +87,45 @@ public void vacuumDatabase() throws SQLException{
             if(ps!=null)
                 ps.close();
         }
+        LOG.info("Found " + activeConglomerates.size() + " active conglomerates.");
 
         //get all the tables from HBaseAdmin
         try {
             Iterable<TableDescriptor> hTableDescriptors = partitionAdmin.listTables();
 
+            if (LOG.isTraceEnabled()) {
+                LOG.trace("Found " + Iterables.size(hTableDescriptors) + " HBase tables.");
+            }
+
             for(TableDescriptor table:hTableDescriptors){
                 try{
                     String[] tableName = parseTableName(table.getTableName());
-                    if (tableName.length < 2) return;
+                    if (tableName.length < 2) {
+                        LOG.warn("Table name doesn't have two components (namespace : name) ignoring: " + table.getTableName());
+                        return;
```

Comment:
If this is not a splice table, should ignore it, instead of returning. This might be the reason why some tables are not cleaned up?